app/views/users/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>I'm a page</h1>
```

Comment:
This is a stub we put up b/c devise wanted a path to route to on sign in. We have a plan to customize our routes so this won't be necessary once we've integrated the venue index template.